docs/index.rst

```diff
@@ -0,0 +1,70 @@
+.. eProsima Discovery Server documentation master file
+
+***************************************
+eProsima Discovery Server Documentation
+***************************************
+
+.. image:: logo.png
+    :height: 80px
+    :width: 80px
+    :align: left
+    :alt: eProsima
+    :target: http://www.eprosima.com/
+
+| `Current DDS-RTPS standard`_ in its section 8.5 specifies a non-centralized, distributed simple discovery mechanism for RTPS. This mechanism was devised to allow
```

Comment:
Line too long. Review all lines that are longer than 120 characters.

Reply:
All lines trimmed to 120 chars.